tests/test_jsonyamlxml_writer.py

```diff
@@ -116,3 +119,30 @@ def write_and_parse_snippet(out_format, snippet, temporary_file_path):
 
     return parsed_snippet
 
+@pytest.mark.parametrize("out_format", ['yaml', 'xml', 'json'])
+def test_files_without_package(temporary_file_path, out_format):
+    document: Document = minimal_document()
+    document.spdx_id = "SPDXRef-DOCUMENT"
+    document.packages[0].spdx_id = "SPDXRef-Package"
+    file: File = minimal_file()
+    document.add_file(file)
+    describes_relationship: Relationship = Relationship("SPDXRef-DOCUMENT DESCRIBES SPDXRef-Package")
+    document.add_relationships(relationship=describes_relationship)
+    contains_relationship: Relationship = Relationship("SPDXRef-Package CONTAINS SPDXRef-File")
+    document.add_relationships(relationship=contains_relationship)
```

Comment:
I don't understand this test. If we are testing a document with files that are not part of a package, we shouldn't have the "package contains file" relationship, and the "document describes package" relationship is superfluous, right? (we could have a "document describes files" relationship instead)

Reply:
Yeah, I started with this test without a package and later added a package without changing the name of the test. So, I changed the test to really test a document without package but with a file.